src/GetText.Extractor/CommandLine/CommandLineOptions.cs

```diff
@@ -78,33 +81,44 @@ private static FileInfo TryParseDefaultTargetFile(ArgumentResult argument)
             return default;
         }
 
-        private static FileInfo TryParseSourceFilePathArgument(ArgumentResult argument)
+        private static IList<FileInfo> TryParseSourceFilePathArgument(ArgumentResult argument)
         {
-            string token = argument.Tokens.Count switch
-            {
-                0 => ".",
-                1 => argument.Tokens[0].Value,
-                _ => throw new InvalidOperationException("Unexpected token count."),
-            };
-            if (File.Exists(token))
+            if (argument.Tokens.Count == 0)
             {
-                return new FileInfo(Path.GetFullPath(token));
```

Comment:
if no -s source argument is given, this now only returns the current folder path without trying to resolve the project/solution file (`TryFindProjectFile`) ? Please restore that piece of functionality.